src/test/runTests.ts

```diff
@@ -16,6 +16,7 @@ async function main() : Promise<void> {
 		const cliPath: string = resolveCliPathFromVSCodeExecutablePath(vscodeExecutablePath);
 		installExtraExtension(cliPath, 'ms-kubernetes-tools.vscode-kubernetes-tools');
 		installExtraExtension(cliPath, 'redhat.java');
+		installExtraExtension(cliPath, 'redhat.vscode-commons');
```

Comment:
Do we still need this if there's no tests?

Reply:
yes, the redhat.vscode-commons is declared as an `extensionDependencies`. Conssequently, to have the VS Code Camel k starting during the tests, this extension needs to be installed.